app/src/main/java/com/telenordigital/protoman/AgreeToUsePossumActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.telenordigital.protoman
+
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager.PERMISSION_GRANTED
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.Toast
+import com.telenor.possumgather.PossumGather
+
+class AgreeToUsePossumActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_agree_to_use_possum)
+
+        val yesButton = findViewById<Button>(R.id.AgreeToUsePossumYesButton)
+        yesButton.setOnClickListener{
+            val possumGather = PossumGather(this,"ProtoMan")
+            if(possumGather.hasMissingPermissions(this)){
+                possumGather.requestNeededPermissions(this)
+            }else{
+                activate()
+            }
+        }
+
+        val laterButton = findViewById<Button>(R.id.AgreeToUsePossumLaterButton)
+        laterButton.setOnClickListener{
+            val intent = Intent(this,EnrollActivity::class.java)
```

Comment:
`val intent = Intent(this,EnrollActivity::class.java)`->
`val intent = Intent(this, EnrollActivity::class.java)`